src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -77,6 +77,14 @@
             doc = "One or more input files (or lists of files) to compare fingerprints for.")
     public List<File> INPUT;
 
+    @Argument(shortName = "I2", optional = true,
+            doc = "One or more input files (or lists of files) to compare fingerprints for. If this option is given " +
+                    "the program to compare each sample in INPUT, with the sample from " +
```

Comment:
"the program compares each sample in INPUT with the sample from INPUT2 that has the same sample ID."

Reply:
done.